pkg/costmodel/cluster.go

```diff
@@ -155,7 +155,7 @@ func ClusterDisks(client prometheus.Client, provider cloud.Provider, start, end
 	ctx := prom.NewNamedContext(client, prom.ClusterContextName)
 	queryPVCost := fmt.Sprintf(`avg(avg_over_time(pv_hourly_cost[%s])) by (%s, persistentvolume,provider_id)`, durStr, env.GetPromClusterLabel())
 	queryPVSize := fmt.Sprintf(`avg(avg_over_time(kube_persistentvolume_capacity_bytes[%s])) by (%s, persistentvolume)`, durStr, env.GetPromClusterLabel())
-	queryActiveMins := fmt.Sprintf(`count(pv_hourly_cost) by (%s, persistentvolume)[%s:%dm]`, env.GetPromClusterLabel(), durStr, minsPerResolution)
+	queryActiveMins := fmt.Sprintf(`avg(kube_persistentvolume_capacity_bytes) by (%s, persistentvolume)[%s:%dm]`, env.GetPromClusterLabel(), durStr, minsPerResolution)
```

Comment:
Any risks to swapping this? And what exactly are the benefits? I can't think of any, in either direction, but want to check in the interest of being super careful.

Reply:
This is in regards to https://github.com/kubecost/kubecost-cost-model/pull/927 I suspect that the use of pv_hourly_cost for our active mins query is causing this log when kubecost is down and not emitting the metric but KSM still is. I might be totally off base on this, but I think this change is low risk and may address the problem